tests/test_image.py

```diff
@@ -1,5 +1,6 @@
 from music21 import *
 
 def test_png():
-    a = converter.parse('tests/sample.musicxml').write('lilypond.png', 'tests/result')
+    a = converter.parse('tests/sample.musicxml').write('lilypond.png')
+    b = converter.parse('tests/sample.musicxml').write('musicxml.png')
     assert 1 == 1
```

Comment:
This line is meaningless, delete. 

Furthermore, I think you need to add a few more test cases for musicxml, we need to test something up to size of several MBs, to ensure that things work.